content/azure-pipelines.yml

```diff
@@ -31,10 +36,17 @@ stages:
               projectKey: "Etch.OrchardCore.ModuleBoilerplate"
               projectName: "Etch.OrchardCore.ModuleBoilerplate"
 
-          - task: tmarkovski.projectversionasvariable.versionintovariable.projectversionasvariable@1
-            displayName: "Get Project Version as variables from Etch.OrchardCore.ModuleBoilerplate.csproj"
+          - task: replacetokens@5
             inputs:
               path: ./Etch.OrchardCore.ModuleBoilerplate.csproj
+              rootDirectory: '$(System.DefaultWorkingDirectory)'
+              targetFiles: |
+                "**/content/README.md"
```

Comment:
This is incorrect, it should target the manifest & csproj file (e.g. https://github.com/EtchUK/Etch.OrchardCore.Fields/blob/main/azure-pipelines.yml#L44).